system-test/README.md

```diff
@@ -1,35 +1,63 @@
-# dm-system-test
+# nnf-system-test
 
-This is a system level (e.g. flux) test suite for verifying NNF Data Movement. The Flux workflow
-manager is used to drive the testing of data movement. Since interaction with Flux is largely at the
+This is a system level (e.g. flux) test suite for verifying NNF Workflows. The Flux workflow manager
+is used to drive the management of NNF Worfklows. Since interaction with Flux is largely at the
 shell level, the tests are written in bash. To drive that, a Bash Automated Testing System (Bats) is
```

Comment:
Drop the "a". It implies that there a multiple things named "Bash Automated Testing System".